java/app/src/main/java/com/fullstorydev/shoppedemo/utilities/Constants.java

```diff
@@ -102,4 +102,13 @@ static public boolean isValidState(int state) {
     public enum Status { SUCCESS, LOADING, ERROR };
 
     public static final String PRODUCT_DATA_FILENAME = "products.json";
-}
\ No newline at end of file
+
+
+    public static final class EVENT_NAMES{
```

Comment:
does this make more sense to be in `FSUtils`?

Reply:
done!